packages/gatsby-source-strapi/index.js

```diff
@@ -0,0 +1 @@
+console.log("noop");
```

Comment:
This can be a comment, not actual code.

Reply:
Well, then lint will crash on [no-empty-file](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-empty-file.md)

Reply:
Damn, the yarn lock is there for my yarn lock fix of last time i did on the main branch, forgot to pull the main branch before making this branch. Will fix it tomorrow, need to run now.

Reply:
> Well, then lint will crash on [no-empty-file](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-empty-file.md)

If we do add this we'll add an ignore.